great_expectations/checkpoint/checkpoint.py

```diff
@@ -99,6 +106,85 @@ def get_substituted_config(
                     self._substituted_config = substituted_config
                 return substituted_config
 
+    def get_runtime_batch_request(
```

Comment:
Is this supposed to be a public method?  If not, let's prefix with an underscore.

Reply:
Yep. Renaming.